### PR TITLE
Minor changes to Portuguese Brazilian strings

### PR DESCRIPTION
**Pull request by bitcoinuser**

### CODE REVIEW

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 1]
```diff
@@ -237,11 +238,11 @@
     <b>Não use esse recurso se você não entender completamente o que ele faz.</b>
   ]]></string>
   <string name="closechannels_force_destination">Os valores serão eventualmente enviados para:</string>
-  <string name="closechannels_force_destination_desc">Este endereço foi originado a partir da sua seed e pertence a você.</string>
+  <string name="closechannels_force_destination_desc">Este endereço tem origem na sua seed e pertence a você.</string>
   <string name="closechannels_force_button">Forçar encerramento de canais</string>
   <string name="closechannels_confirm_dialog_message">Tem certeza de que deseja continuar?</string>
   <string name="closechannels_message_error">Ocorreu um problema. Tente novamente.</string>
-  <string name="closechannels_message_done">Bitcoins enviados.</string>
+  <string name="closechannels_message_done">bitcoins enviados.</string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:245]:
```suggestion
  <string name="closechannels_message_done">Fundos enviados.</string>
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 2]
```diff
@@ -142,7 +143,7 @@
   <string name="send_pay_button">Pagar</string>
   <string name="send_swap_in_progress">Preparando transação…</string>
   <string name="send_swap_error">A troca para rede Bitcoin falhou. Tente novamente.</string>
-  <string name="send_swap_required_desc">Este é um endereço Bitcoin, o pagamento será realizado usando uma troca.</string>
+  <string name="send_swap_required_desc">Este é um endereço bitcoin, o pagamento será realizado usando uma troca.</string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:146]:
```suggestion
  <string name="send_swap_required_desc">Este é um endereço Bitcoin, o pagamento será realizado usando uma troca.</string>
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 3]
```diff
@@ -93,12 +93,12 @@
...
 
-  <string name="receive_swap_in_button">Mostrar endereço Bitcoin</string>
+  <string name="receive_swap_in_button">Mostrar endereço bitcoin</string>
   <string name="receive_swap_in_error_message">Ocorreu um erro ao trocar para uma transação da rede Bitcoin…\n\nTente novamente.</string>
-  <string name="receive_swap_in_progress_message">Solicitando um endereço Bitcoin…</string>
+  <string name="receive_swap_in_progress_message">Solicitando um endereço bitcoin…</string>
   <string name="receive_swap_in_disclaimer_title">Receber pela rede Bitcoin</string>
   <string name="receive_swap_in_disclaimer_message"><![CDATA[
-  Um endereço Bitcoin padrão será exibido a seguir.
+  Um endereço bitcoin padrão será exibido a seguir.
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:101]:
```suggestion
  Um endereço Bitcoin padrão será exibido a seguir.
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 4]
```diff
@@ -93,12 +93,12 @@
   <string name="receive_power_save_disclaimer">Você está no modo de economia de energia. Para receber o pagamento, verifique se o aplicativo pode ser executado em segundo plano.</string>
   <string name="receive_withdraw_button">Usar link LNURL</string>
 
-  <string name="receive_swap_in_button">Mostrar endereço Bitcoin</string>
+  <string name="receive_swap_in_button">Mostrar endereço bitcoin</string>
   <string name="receive_swap_in_error_message">Ocorreu um erro ao trocar para uma transação da rede Bitcoin…\n\nTente novamente.</string>
-  <string name="receive_swap_in_progress_message">Solicitando um endereço Bitcoin…</string>
+  <string name="receive_swap_in_progress_message">Solicitando um endereço bitcoin…</string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:98]:
```suggestion
  <string name="receive_swap_in_progress_message">Solicitando um endereço Bitcoin…</string>
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 5]
```diff
@@ -93,12 +93,12 @@
   <string name="receive_power_save_disclaimer">Você está no modo de economia de energia. Para receber o pagamento, verifique se o aplicativo pode ser executado em segundo plano.</string>
   <string name="receive_withdraw_button">Usar link LNURL</string>
 
-  <string name="receive_swap_in_button">Mostrar endereço Bitcoin</string>
+  <string name="receive_swap_in_button">Mostrar endereço bitcoin</string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:96]:
```suggestion
  <string name="receive_swap_in_button">Mostrar endereço Bitcoin</string>
```

File: app/src/main/res/values-pt-rBR/strings.xml (hunk 2 above)
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:146]:
```suggestion
  <string name="send_swap_required_desc">Este é um endereço Bitcoin, o pagamento será realizado usando uma troca.</string>
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 6]
```diff
@@ -221,10 +223,10 @@
   <string name="closechannels_channels_none">No momento, você não possui canais que possam ser esvaziados.\n\nEm \'Lista de canais\' você pode verificar o estado em que se encontram os seus canais.</string>
   <string name="closechannels_mutual_title">Esvaziar carteira</string>
   <string name="closechannels_mutual_instructions"><![CDATA[
-    Use este recurso para esvaziar sua carteira enviando o saldo para um endereço Bitcoin.
+    Use este recurso para esvaziar sua carteira enviando o saldo para um endereço bitcoin.
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:226]:
```suggestion
    Use este recurso para esvaziar sua carteira enviando o saldo para um endereço Bitcoin.
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 7]
```diff
@@ -221,10 +223,10 @@
...
   ]]></string>
   <string name="closechannels_mutual_input_instructions">Os valores podem ser enviados para uma carteira Bitcoin. Antes de enviar verifique se o endereço está correto.</string>
-  <string name="closechannels_mutual_input_hint">Insira um endereço Bitcoin</string>
+  <string name="closechannels_mutual_input_hint">Insira um endereço bitcoin</string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:229]:
```suggestion
  <string name="closechannels_mutual_input_hint">Insira um endereço Bitcoin</string>
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 8]
```diff
@@ -124,12 +124,13 @@
...
   <string name="send_use_all_balance">Enviar tudo</string>
   <string name="send_loading">Verificando pagamento…</string>
-  <string name="send_invalid_pr_generic">Este pagamento não é válido. Você pode estar lendo um código que não representa uma cobrança de Bitcoins.\n\nTente novamente.</string>
+  <string name="send_invalid_pr_generic">Este pagamento não é válido. Você pode estar lendo um código que não representa uma cobrança de bitcoins.\n\nTente novamente.</string>
   <string name="send_amount_error">Insira uma quantia válida.</string>
   <string name="send_amount_error_balance">O valor excede o seu saldo.</string>
   <string name="send_amount_error_swap_out_too_small">Mín. 10000 sat para pagamentos pela rede Bitcoin.</string>
   <string name="send_destination_label">Enviar para</string>
   <string name="send_description_label">Desc.</string>
+  <string name="send_chain_fees_edit">Ajustar taxas</string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:133]:
```suggestion
  <string name="send_chain_fees_edit">Ajustar taxa</string>
```

**Comment by bitcoinuser** [comment]
Hi @dpad85 

It's ok to be merged. Thanks.

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 9]
```diff
@@ -124,12 +124,13 @@
...
   <string name="send_use_all_balance">Enviar tudo</string>
   <string name="send_loading">Verificando pagamento…</string>
-  <string name="send_invalid_pr_generic">Este pagamento não é válido. Você pode estar lendo um código que não representa uma cobrança de Bitcoins.\n\nTente novamente.</string>
+  <string name="send_invalid_pr_generic">Este pagamento não é válido. Você pode estar lendo um código que não representa uma cobrança de bitcoins.\n\nTente novamente.</string>
   <string name="send_amount_error">Insira uma quantia válida.</string>
   <string name="send_amount_error_balance">O valor excede o seu saldo.</string>
   <string name="send_amount_error_swap_out_too_small">Mín. 10000 sat para pagamentos pela rede Bitcoin.</string>
   <string name="send_destination_label">Enviar para</string>
   <string name="send_description_label">Desc.</string>
+  <string name="send_chain_fees_edit">Ajustar taxa</string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:133]:
```suggestion
  <string name="send_chain_fees_edit">Ajustar taxas</string>
```

File: app/src/main/res/values-pt-rBR/strings.xml (hunk 8 above)
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:133]:
```suggestion
  <string name="send_chain_fees_edit">Ajustar taxa</string>
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 10]
```diff
@@ -83,7 +83,7 @@
   <string name="receive_amount_label">Quantidade (opcional)</string>
   <string name="receive_desc_label">Descrição do pagamento (opcional):</string>
   <string name="receive_default_desc">Pagamento Phoenix</string>
-  <string name="receive_generate_button">Cobrar</string>
+  <string name="receive_generate_button">Solicitar</string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:86]:
```suggestion
  <string name="receive_generate_button">Cobrar</string>
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 11]
```diff
@@ -228,20 +230,20 @@
...
   <string name="closechannels_force_instructions"><![CDATA[
-    Encerramento forçado permite fechar unilateralmente seus canais.
+    O encerramento forçado permite fechar seus canais de forma unilateral.
     <br/><br/>
     Esta função não é &quot;um botão mágico que resolve tudo&quot;; foi inserida como uma medida de segurança e <b>só deve ser usada em casos extremos</b>. Por exemplo, se seu parceiro (ACINQ) desaparecer permanentemente, impedindo que você gaste seus bitcoins. Em todos os outros casos, <b>se você tiver problemas com a Phoenix, entre em contato com o suporte</b>.
     <br/><br/>
     Forçar o encerramento de canais irá lhe custar dinheiro (para cobrir as taxas da rede Bitcoin) além de fazer com que seus fundos fiquem bloqueados por dias. <b>Não desinstale o aplicativo até que seus canais estejam totalmente encerrados ou você perderá bitcoins.</b>
     <br/><br/>
-    <b>Não use esse recurso se você não entender completamente o que ele faz.</b>
+    <b>Não use esse recurso se você não compreende exatamente o que ele faz.</b>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:239]:
```suggestion
    <b>Não use esse recurso se você não compreender exatamente o que ele faz.</b>
```

**Comment by dpad85** [comment]
Thanks for the PR @bitcoinuser 👍

**Pull request merged**